src/components/docs-anchor/index.tsx

```diff
@@ -4,11 +4,11 @@ import { useCurrentProduct } from 'contexts'
 import InlineLink from 'components/inline-link'
 import s from './docs-anchor.module.css'
 
-const DocsAnchor: React.FC<JSX.IntrinsicElements['a']> = ({
+const DocsAnchor = ({
   href,
   children,
   ...rest
-}) => {
+}: JSX.IntrinsicElements['a']) => {
```

Comment:
These changes fix a typedef warning.

Reply:
FWIW, the type was correct. I just don't think the lint rule has enough type information like the TS compiler does to deduce the correct type here. We also technically lose the return type from this change.

Reply:
Does the inferred return type of `JSX.Element` still suffice?

Reply:
Should be fine!